codalab/model/bundle_model.py

```diff
@@ -1616,6 +1616,14 @@ def add_worksheet_items(self, worksheet_uuid, items, after_sort_key=None, replac
             if len(items) == 0:
                 # Nothing to insert, return
                 return
+
+            # if after_sort_key not specified, insert the bundle to the end of the worksheet
```

Comment:
What was the previous behavior if after_sort_key was not specified?

Reply:
it's just null